src/sentry/runner/commands/run.py

```diff
@@ -303,6 +303,12 @@ def cron(**options):
     type=click.Choice(["earliest", "latest"]),
     help="Position in the commit log topic to begin reading from when no prior offset has been recorded.",
 )
+@click.option(
+    "--topic-config",
+    default=None,
+    type=click.Choice(["events", "transactions"]),
+    help="The topic to subscribe to. This is the key of the config in KAFKA_TOPICS settings",
```

Comment:
Also passing the topic instead of the event type is the only way to have a safe rollout in that we can rollout separately the post process consumer, which has to happen first, from the process of producing to the new topic.